_posts/2019-11-14-april-2020-workshop.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: The First US-RSE Association Workshop
+posted_by: Ian Cosden
+tags: [workshop]
+---
+
+We are excited to announce that through a generous grant from the Alfred P. Sloan Foundation, we will be able to hold the first US-RSE Association community building workshop, April 21-22, 2020 in Princeton, NJ.
+This won't be an attempt to be an RSE conference, but a workshop focused on planning the best path forward to grow the US-RSE Association.
+
+
+The workshop will have breakout sessions focused on a number of different topics pertinent to the US-RSE community.
+We’ll be working together to generate material/content for the us-rse.org website that promotes and supports current/future RSEs in alignment with the [mission](https://us-rse.org/mission/) of the US-RSE Association, and to plan follow-up activities to grow the community.  
+More specific workshop information and details will follow in the coming weeks.
+
+You can find out more and express interest in attending by filling out the form here: https://forms.gle/tzronZUGdUTT9zzb9.
+Even though we have to keep this workshop small, remember that there are many ways to get more involved in the community and help grow the US-RSE Association.
```

Comment:
Any reason to not do [the form here](https://forms.gle/tzronZUGdUTT9zzb9) ?

Reply:
It was an intentional cop out to make it more noticeable.  Is it too ugly?

Reply:
I'm usually in favor of hiding the URLs, maybe make it more description (and bold)?

... express interest in attending by filling out the [**US-RSE Workshop form**](https://forms.gle/tzronZUGdUTT9zzb9).

Reply:
OK Done